### ISSUE TITLE
Update TLDR explainer

### ISSUE THREAD

**Issue by TheJaredWilcurt** [TheJaredWilcurt/vue-doxen#61]
Swap out Patrick for something like this:

```

TLDR

              Put documentation in your components
                      /               \
 _________________________         _________________________
|                         |       |                         |
|<script>                 |       |<script setup>           |
|export default {         |       |defineOptions({          |
|  name: 'MyButton',      |       |  name: 'MyCheckbox',    |
|  description: '...',    |       |  description: '...'     |
|  props: {               |       |});                      |
|    selected: {          |       |defineProps({            |
|      type: Boolean,     |       |  selected: {            |
|      description: '...' |       |    type: Boolean,       |
|    }                    |       |    description: '...'   |
|  }                      |       |  }                      |
|};                       |       |});                      |
|</script>                |       |</script>                |
|_________________________|       |_________________________|

                      \               /
               Pass components into Vue-Doxen
                             |
                ___________________________
               |                           |
               |<template>                 |
               |  <VueDoxen                |
               |    v-model="selectedDemo" |
               |    :demos="{              |
               |      MyButton,            |
               |      MyCheckbox           |
               |    }"                     |
               |  />                       |
               |</template>                |
               |___________________________|

```

Should still work on all screen sizes. This is probably the most important part of the site to work well on very tiny screens, since people may be giving the project a first impression on a device they wouldn't use for development.
